openapi-filter.js

```diff
@@ -30,20 +31,22 @@ let argv = require('yargs')
     .number('lineWidth')
     .alias('l', 'lineWidth')
     .describe('lineWidth','max line width of yaml output')
-    .default('lineWidth',-1)
+    .default('lineWidth',80)
+    .alias('i','inverse')
```

Comment:
You seem to have reinstated this duplicate line when a merge went wrong.  Can you remove?